os_brick/initiator/connectors/remotefs.py

```diff
@@ -105,16 +115,24 @@ def connect_volume(self, connection_properties):
         return {'path': path}
 
     @utils.trace
+    @synchronized('disconnect_volume')
     def disconnect_volume(self, connection_properties, device_info,
                           force=False, ignore_errors=False):
-        """No need to do anything to disconnect a volume in a filesystem.
+        """Ensure that the filesystem containing the volume is unmounted.
```

Comment:
Maybe add a note here that this is only for those volumes that are in separate mounted filesystems?

Reply:
Sure, I'll update the comment, thank you!

Reply:
Fixed by cf8589323c0169e654c82fe83ec000adf97f783c